### DISCUSSION TITLE
115网盘分享，    alist-android-arm版本报错

### DISCUSSION THREAD

**Discussion by nxjonson** [AlistGo/alist#6407] [Q&A] [answered]
failed get objs: failed to list objs: json: cannot unmarshal number 2156002676509098546 into Go struct field ShareFile.data.list.cid of type int

**Comment by nxjonson** [comment]
分享网址https://115.com/s/sw68lo93nnw?password=kb39&#

**Reply by SheltonZhu** [reply]
是挂载的时候报错么？还有别的信息么？如操作系统的架构 你这个目测是因为 32位操作系统， 2156002676509098546 转int超出了int32的最大值导致的

我用amd64 linux 挂载你这个链接是成功的

**Reply by nxjonson** [reply, marked as answer]
alist-android-arm64也是成功的
